content/hi/docs/tutorials/hello-minikube.md

```diff
@@ -0,0 +1,306 @@
+---
+title: हेलो मिनीक्यूब
+content_type: tutorial
+weight: 5
+menu:
+  main:
+    title: "शुरू करते हैं"
+    weight: 10
+    post: >
+      <p>तो क्या आप तैयार हैं? एक सामान्य ऐप चलने के लिए एक साधारण कुबेरनेट्स क्लस्टर बनाएं।</p>
+card:
+  name: tutorials
+  weight: 10
+---
+
+<!-- overview -->
+
+यह ट्यूटोरियल आपको मिनिक्यूब और काटाकोडा का उपयोग करते हुए 
+कुबेरनेट्स पर एक साधारण ऐप चलाने का तरीका दिखाता है।
+काटाकोडा आपके ब्राउज़र पर मुफ़्त कुबेरनेट्स वातावरण प्रदान करता है।
+
+{{< note >}}
+यदि आपने अपने स्थानीय सिस्टम पर मिनीक्यूब स्थापित किया है तो आप इस ट्यूटोरियल का अनुसरण कर सकते हैं।
+स्थापाना निर्देश के लिए [मिनीक्यूब पृष्ट](https://minikube.sigs.k8s.io/docs/start/) देखें। 
+{{< /note >}}
+
+## {{% heading "objectives" %}}
+
+* मिनीक्यूब में एक नमूना एप्लीकेशन डेप्लॉय करें।
+* ऐप को चलाएं।
+* एप्लिकेशन लॉग देखें।
+
+## {{% heading "prerequisites" %}}
+
+
+यह ट्यूटोरियल एक कंटेनर इमेज प्रदान करता है जो सभी अनुरोधों को प्रतिध्वनित करने के लिए NGINX का उपयोग करता है।
+
+
+
+<!-- lessoncontent -->
+
+## एक मिनीक्यूब क्लस्टर बनाएं
+
+1. **Launch Terminal** पर क्लिक करें।
+
+    {{< kat-button >}}
+
+{{< note >}}
+यदि आपने स्थानीय रूप से मिनीक्यूब स्थापित किया है, तो `minikube start` चलाएँ। इससे पहले कि आप `minikube dashboard` चलाएं, आपको एक नया टर्मिनल खोलना चाहिए, वहां `minikube dashboard` शुरू करना चाहिए, और फिर मुख्य टर्मिनल पर वापस जाना चाहिए।
+{{< /note >}}
+
+2. ब्राउज़र में कुबेरनेट्स डैशबोर्ड खोलें:
+
+    ```shell
+    minikube dashboard
+    ```
+
+3. केवल काटाकोडा वातावरण के लिए: टर्मिनल फलक के शीर्ष पर, प्लस(+) चिह्न पर क्लिक करें, और फिर **Select port to view on Host 1** क्लिक करें।
```

Comment:
करें,  फिर delivers same meaning

Reply:
My opinion would be to keep "फिर" in the sentence to make things more clear. English version also uses "then" which is same as using "फिर"